marklogic-data-hub-central/ui-custom/src/components/Membership/Membership.tsx

```diff
@@ -0,0 +1,71 @@
+import React, {useContext} from 'react';
+import "./Membership.scss";
+import {Check, X} from 'react-bootstrap-icons';
+import _ from "lodash";
+import {getValByConfig} from '../../util/util';
+
+type Props = {
+  data?: any;
+  config?: any;
+};
+
+/**
+ * Component for showing one or more values for a Membership view.
+ *
+ * @component
+ * @prop {object} data - Data payload.
+ * @prop {object} config  Membership configuration object.
+ * @example
+ * {
+      "component": "Relationships",
+      "config": {
+        "arrayPath": "person.memberships.membership",
+        "iconSize": 35,
+        "lists": [
+          "list1",
+          "list2",
+          "list3",
+          "list4",
+          "list5",
+          "list6",
+          "list7",
+          "list8",
+          "list9",
+          "list10",
+          "list11",
+          "list12"
+        ]
+      }
+    }
+ */
+const Membership: React.FC<Props> = (props) => {
+  const {data, config} = props;
+  const iconSize = config.iconSize ? config.iconSize : 30
```

Comment:
Allow configuration of item container style:
```
let membershipStyle: any = props.style ? props.style : props.config?.style ? props.config.style : {};
```
Then apply the style in the item DIV:
```
<div key={index} className={classes} style={membershipStyle}>
```